src/common/exif.cc

```diff
@@ -828,7 +851,8 @@ void dt_exif_img_check_additional_tags(dt_image_t *img, const char *filename)
     if(!exifData.empty())
     {
       _check_usercrop(exifData, img);
-      _check_dng_opcodes(exifData, img);
+      _check_dng_opcodes2(exifData, img);
+      _check_dng_opcodes3(exifData, img);
```

Comment:
Beware that this can introduce additional overheard in imageio_rawspeed re https://github.com/darktable-org/darktable/pull/12844